packages/docs-browser/src/components/Action/actions/Move.js

```diff
@@ -65,7 +65,7 @@ export const MoveAction = ({
       selectionStyle="none"
       searchFormType="none"
       disableViewPersistor={true}
-      listFormName={parent === null ? 'Move_root_docs_list_item' : 'Move_docs_list_item'}
+      getListFormName={(parent, _) => parent === null ? 'Move_root_docs_list_item' : 'Move_docs_list_item'}
```

Comment:
getListFormName={parent => parent === null ? 'Move_root_docs_list_item' : 'Move_docs_list_item'}
should be fine
underscore for a not used parameter is not a thing we are doing